ui/pages/SummaryData/components/Hpo.jsx

```diff
@@ -96,10 +121,23 @@ class Hpo extends React.PureComponent {
         ))}
         <Divider />
         {terms.length > 0 && (
-          <Header size="small">
-            <Header.Content>{`${families.size} Families, ${data.length} Individuals: `}</Header.Content>
-            <HorizontalSpacer width={10} />
-            <ButtonLink as={NavLink} disabled={!searchHref} target="_blank" to={searchHref}>Variant Search</ButtonLink>
+          <Header size="medium">
+            <Header.Content>{`${numFamilies} Families, ${data.length} Individuals`}</Header.Content>
+            <Header.Subheader>
+              {genomeSearchPaths.map(([genomeVersion, searchPath]) => (
```

Comment:
I understand you break down the search into genome version-based searches. It makes the search smaller to solve the issue. Can we break down the search further? I'm asking because per genome version search could still be big.

Reply:
I did not break it down because of size, I broke it down because seqr throws an error if you search across multiple genome versions and will not let you do the search. The restructuring of the URL to be grouped by project guid, combined with the existing max families for search, will keep the URL short enough